cmd/branchProtection.go

```diff
@@ -52,7 +52,7 @@ func (verifier BranchProtectionVerifier) CheckIfBranchProtectionIsApplied(fix bo
 	repo := verifier.getRepo()
 	branch := *repo.DefaultBranch
```

Comment:
```suggestion
	defaultBranch := *repo.DefaultBranch
```
(also other occurrences)